lib/service/commands/cleanup.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Service
+  module Commands
+    module Cleanup
+      module_function
+
+      def run
+        Service::ServicesCli.cleanup
```

Comment:
Is the plan to move a bunch of the logic into this method?

Reply:
For cleanup I doubt there would be much. But I expect restart to eventually just be `start && stop`

Reply:
Yeh, I guess I'm thinking right now "why not just call `Service::ServicesCli.cleanup` directly?". I'd presume "that logic will be moved here" but if not: I'm not sure about this refactoring.

Reply:
Yeah, I'd like to move as much of the shared logic there as possible. I believe that's also how bundle does it right?